### PR TITLE
Database Updates for Sponsors

### PR DESCRIPTION
**Pull request by nicktodd12**
Create database and relevant model classes for storing and retrieving sponsor information using SQLDelight.

### CODE REVIEW

File: sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/SponsorModel.kt [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
-        proc(backgroundSuspend {
-            JSON.nonstrict.parse(SponsorGroup.serializer().list, AppContext.sponsorJson)
-        })
+class SponsorModel : BaseQueryModelView<Sponsor, List<SponsorGroupDbItem>>(
+        sponsorQueries.selectAll(),
+        {
+            val sponsors = it.executeAsList()
+            val groupList = mutableMapOf<String, ArrayList<Sponsor>>()
+
+            for(sponsor in sponsors) {
```
[the remark this answers is not in the document]
Comment (nicktodd12) [reply sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/SponsorModel.kt]: Pretty cool, I didn't know Kotlin had so many collections functions.  Managed to simplify this quite a bit.  Thanks for the suggestion.

File: sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt [hunk 2]
```diff
@@ -153,7 +157,7 @@ object AppContext {
 
     private fun storeAll(networkSponsorJson: String, networkSpeakerJson: String, networkSessionJson: String) {
         appSettings.putString(SPONSOR_JSON, networkSponsorJson)
```
Comment (samhill303) [inline sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt]: I don't think we need to put the sponsors into app settings anymore right?

Comment (nicktodd12) [reply sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt]: You're correct, I meant to get rid of this when I switched over the iOS side.  It's removed now.

File: sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/db/SponsorGroupDbItem.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package co.touchlab.sessionize.db
+
+import co.touchlab.droidcon.db.Sponsor
+
+data class SponsorGroupDbItem(val groupName: String, val sponsors: List<Sponsor>)
```
Comment (kpgalligan) [inline sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/db/SponsorGroupDbItem.kt]: I think these data objects need a rename, but we should do that post merge (and "to what" I'm not sure, but seems a little verbose)

**Pull request merged**